src/image-viewer.js

```diff
@@ -339,17 +340,27 @@ export class ImageViewer {
       const deltaX = this.viewerBox_.width / 2 - e.data.clientX;
       const deltaY = this.viewerBox_.height / 2 - e.data.clientY;
       this.onZoom_(newScale, deltaX, deltaY, true).then(() => {
-        return this.onZoomRelease_(0, 0, 0, 0, 0, 0);
+        return this.onZoomRelease_();
       });
     });
     gestures.onGesture(TapzoomRecognizer, e => {
-      this.onZoomInc_(e.data.centerClientX, e.data.centerClientY,
+      event.preventDefault();
+      this.onTapZoom_(e.data.centerClientX, e.data.centerClientY,
           e.data.deltaX, e.data.deltaY);
       if (e.data.last) {
-        this.onZoomRelease_(e.data.centerClientX, e.data.centerClientY,
+        this.onTapZoomRelease_(e.data.centerClientX, e.data.centerClientY,
             e.data.deltaX, e.data.deltaY, e.data.velocityY, e.data.velocityY);
       }
     });
+    gestures.onGesture(PinchRecognizer, e => {
+      // To disable iOS 10 Safari default pinch zoom
+      event.preventDefault();
```

Comment:
I suggest removing `disableTouchZoom` from `enterOverlayMode` (a new PR might be better, maybe an issue to do it later).

Reasoning is:
1- the old `disableTouchZoom`  don't work on iOS anymore
2- Disabling touchzoom for every overlay is not the right thing to do. It only makes sense for image viewer and similar cases not every lightbox.

Reply:
Going to investigate this separately.